src/ingest_llm_as/models/knowledge_digest.py

```
@@ -0,0 +1,70 @@
+        metadata={
+            "platform": platform,
+            "message_count": message_count,
+            "vault_path": vault_filepath,
+            "original_url": raw_payload.get("url")
+        },
+        tags=["ai_conversation", platform.lower()],
+        references={
+            "vault_file": vault_filepath,
+            "url": raw_payload.get("url")
+        },
+        captured_at=captured_at or datetime.utcnow()
```

Ruff (0.14.11)

`datetime.datetime.utcnow()` used

(DTZ003)